ktxspec.adoc

```diff
@@ -1014,6 +1014,47 @@ Still, `KHR_DF_PRIMARIES_BT709` / `KHR_DF_PRIMARIES_SRGB` is recommended
 for standard dynamic range, standard gamut images.
 ====
 
+==== Use of Transfer Functions
+The majority of transfer functions listed in Chapter 13 _Transfer
+functions_ of <<KDF13>>, with corresponding enumerators in the
+`khr_df_transfer_e` enumeration of `khr_df.h`, have a linear
+opto-optical transfer function. That is the opto-electrical transfer
+function (OETF) is the inverse of the electro-optical transfer
+function (EOTF). When a KTX file has one of these transfer functions
+
+* applications should use the EOTF to decode prior to sampling.
```

Comment:
Since the next line starts with a lower-case letter, this line should end with a semicolon. Same comment for other two similar subsections.

Reply:
Fixed in [813bd67](https://github.com/KhronosGroup/KTX-Specification/pull/222/commits/813bd67931c1713f34a58dff71f2505c36d17798).